packages/manager/apps/web/client/app/private-database/order/clouddb/private-database-order-clouddb.controller.js

```diff
@@ -1,93 +1,100 @@
 import get from 'lodash/get';
-import orderBy from 'lodash/orderBy';
-
-import { PRODUCT_NAME } from './private-database-order-clouddb.constants';
+import {
+  pricingConstants,
+  workflowConstants,
+} from '@ovh-ux/manager-product-offers';
+import {
+  PRESELECTED_DB_CATEGORY,
+  PRODUCT_NAME,
+} from './private-database-order-clouddb.constants';
+import {
+  DATACENTER_CONFIGURATION_KEY,
+  DB_OFFERS,
+  ENGINE_CONFIGURATION_KEY,
+} from '../../../hosting/database/order/public/hosting-database-order-public.constants';
+import { DATABASES_TRACKING } from '../../../hosting/hosting.constants';
 
 export default class PrivateDatabaseOrderCloudDbCtrl {
   /* @ngInject */
-  constructor($filter, $translate, PrivateDatabaseOrderCloudDb) {
+  constructor(
+    $filter,
+    $translate,
+    PrivateDatabaseOrderCloudDb,
+    $anchorScroll,
```

Comment:
**nitpick:** This dependency is not used. Can you remove it?

Reply:
I don't understand since it is used.

![image](https://github.com/ovh/manager/assets/8767496/9607b74c-c81a-4c16-8f52-7683818d9115)

Reply:
Ah ok got it, it was for filtrer and translate, thks!